asciidoc-confluence-publisher-doc/etc/docs/00-index/06-images.adoc

```diff
@@ -69,5 +81,5 @@ This image has a link image:../images/frisbee.png[width=16, height=16, link=http
 
 [WARNING]
 ====
-Image captions and images from URLs are currently not supported.
+Image captions are currently not supported.
```

Comment:
Actually, the warning is completely obsolete, as image captions are already supported on master. Would you mind piggybacking this small change on your PR and remove the whole warning? Thank you!